types/webxr/index.d.ts

```diff
@@ -227,7 +227,7 @@ export interface XRSession {
      * canceling the callback using cancelAnimationFrame(). This method is comparable
      * to the Window.requestAnimationFrame() method.
      */
-    requestAnimationFrame: XRFrameRequestCallback;
+    requestAnimationFrame: (callback: XRFrameRequestCallback) => number;
```

Comment:
It's odd my local copy already has this change... so odd.

Can you add something like this to the test?

```
...
.then(rs => {
                space = rs as XRReferenceSpace;
...
                const loop: XRFrameRequestCallback = (now: number, frame: XRFrame) => { }
                const n = session.requestAnimationFrame(loop);
                session.cancelAnimationFrame(n);
...
```

So we can lock this in place a bit better?

Reply:
Added! I'm not sure what the etiquette is on marking conversations resolved (I know some repos prefer leaving that to the original commenter as a confirmation that the issue was truly addressed), so I'll leave this open and re-request review. Thanks!